Formula/l/livekit-cli.rb

```diff
@@ -19,12 +19,13 @@ class LivekitCli < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/livekit-cli"
+    ldflags = "-s -w"
+    system "go", "build", *std_go_args(ldflags:, output: bin/"lk"), "./cmd/lk"
   end
 
   test do
-    output = shell_output("#{bin}/livekit-cli create-token --list --api-key key --api-secret secret")
+    output = shell_output("#{bin}/lk create-token --list --api-key key --api-secret secret")
```

Comment:
```suggest
   output = shell_output("#{bin}/lk token create --list --api-key key --api-secret secret")
```

Reply:
The legacy command should still work in this version, but is considered deprecated and will be removed in the future.

Reply:
@chenrui333 We also wish to symlink `livekit-cli`, so as not to break existing scripts and automations. Would you prefer to reopen the other PR or make this update yourself?

Reply:
yeah, I will update in here, seems reasonable.